llvm/docs/CommandGuide/llvm-objcopy.rst

```diff
@@ -120,7 +120,8 @@ multiple file formats.
 .. option:: --remove-symbol-prefix <prefix>
 
  Remove ``<prefix>`` from the start of every symbol name. No-op for symbols that do
- not start with ``<prefix>``.
+ not start with ``<prefix>``. Can be specified multiple times to remove multiple
```

Comment:
I am thinking of some variants, e.g.

"The option can be specified more than once." Each candidate can use exactly the same message.

"Multiple occurrences are feasible."

Reply:
We need to distinguish between two behaviors. 1. The option can be specified multiple times, the last one wins. 2. The option can be specified N times to perform N actions simultaneously.

Maybe: "Can be specified multiple times, each instance is processed separately"?

Reply:
Personally, I don't think we need to identify case 1. Case 1 should be considered the default behaviour in LLVM tool command-line handling so doesn't need calling out.

Reply:
My point was that "Multiple occurrences are feasible" could be either 1. or 2.